examtex

```diff
@@ -37,11 +37,11 @@ if [ "$err" != "" ]; then
 	echo "$err"
 # compile pdf from tex
 elif [ $to_pdf = 'true' -a "$err" = "" ]; then
-	cd "$(dirname "$1")"
-	latexmk -pdf -quiet "${1%.*}-EXAM.tex"
-	latexmk -pdf -quiet "${1%.*}-KEY.tex"
+	cd $cwd
+	latexmk -pdf -f -quiet "${1%.*}-EXAM.tex"
```

Comment:
was getting weird build errors and it wouldn't complete unless i had this `-f` flag

Reply:
try compiling to tex and then using sublime to compile to pdf. Tryna see if the program is generating wrong tex files..